### PR TITLE
17 unit tests for each mvp component and workflow

### LINKED ISSUE #17: unit tests for each MVP component and workflow

### PR DESCRIPTION
**Pull request by wyli**
Fixes #17.

### Description
unit tests -- mean dice and stats event handlers
integration test -- sliding window inference

### Status
**Ready**

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New tests added to cover the changes
- [x] Docstrings/Documentation updated

### CODE REVIEW

File: monai/handlers/mean_dice.py [hunk 1]
```diff
@@ -39,12 +39,15 @@ def __init__(
         Args:
             include_background (Bool): whether to include dice computation on the first channel of the predicted output.
             to_onehot_y (Bool): whether to convert the output prediction into the one-hot format.
```
Comment (madil90) [inline monai/handlers/mean_dice.py]: should mention default value in docstring.

File: tests/integration_sliding_window.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        dimensions=3,
+        in_channels=1,
+        num_classes=1,
+        channels=(4, 8, 16, 32),
+        strides=(2, 2, 2),
+        num_res_units=2,
+    )
+    roi_size = (16, 32, 48)
+    sw_batch_size = 2
+    device = torch.device('cpu:0')
```
Comment (madil90) [inline tests/integration_sliding_window.py]: Overrides device parameter in arguments.

File: tests/test_handler_mean_dice.py [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+import unittest
+
+import torch
+from parameterized import parameterized
+
+from monai.handlers.mean_dice import MeanDice
+
+TEST_CASE_1 = [{'to_onehot_y': True, 'mutually_exclusive': True}, 0.75]
+TEST_CASE_2 = [{'include_background': False, 'to_onehot_y': False, 'mutually_exclusive': False}, 0.8333333]
+
```
Comment (madil90) [inline tests/test_handler_mean_dice.py]: Should have corner case testing like passing None, shapes not matching etc.

File: monai/handlers/mean_dice.py [hunk 4]
```diff
@@ -39,12 +39,15 @@ def __init__(
...
-            logit_thresh (Float): the threshold value to round value to 0.0 and 1.0, default is 0.5.
+            logit_thresh (Float): the threshold value to round value to 0.0 and 1.0.
             add_sigmoid (Bool): whether to add sigmoid function to the output prediction before computing Dice.
```
Comment (madil90) [inline monai/handlers/mean_dice.py]: Default value for add_sigmoid is important too
